_posts/2021-03-04-geoserver-2-19-rc-released.md

```diff
@@ -0,0 +1,195 @@
+---
+author: Andrea Amie, Jody Garnett
+date: 2021-03-04
+layout: post
+title: GeoServer 2.19-RC Released
+categories:
+- Announcements
+tags:
+- Release Candidate
+---
+
+We are happy to announce GeoServer [2.19-RC](/release/2.19-RC/) release candidate is available for testing. Downloads are available ([zip](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/geoserver-2.19-RC-bin.zip/download) and [war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/geoserver-2.19-RC-war.zip/download)) along with [docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/geoserver-2.19-RC-htmldoc.zip/download) and [extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/extensions/).
+
+This is a GeoServer release candidate made in conjunction with GeoTools 25-RC and GeoWebCache 1.19-RC.
+
+  * Release candidates are a community building exercise and are not intended for production use.
+  * We ask the community (everyone: individuals, organizations, service providers) to download and thoroughly test this release candidate and report back.
+  * Participating in testing release candidates is a key expectation of our [open source social contract](http://www.ianturton.com/talks/foss4g.html#/). We make an effort to thank each person who tests in our release announcement and project presentations!
+  * GeoServer [commercial service providers](http://geoserver.org/support/) are fully expected to test on behalf of their customers.
```

Comment:
would this be better as "are expected to fully test on behalf of their customers."?

Reply:
That would have a slightly different meaning (ie “test everything”). I am trying to say “we expect you to test”.

Reply:
OK.